### PR TITLE
Doc - using managedclusteraddon to deploy submariner-addon

### PR DESCRIPTION
**Pull request by skeeey**
Signed-off-by: liuwei <liuweixa@redhat.com>

### CODE REVIEW

**Comment by skeeey** [comment]
refer to https://github.com/open-cluster-management/backlog/issues/6440

File: docs/submarinerAddon.md [hunk 1]
```diff
@@ -26,19 +26,23 @@ There are some prerequisites for the managed clusters which are going to run `Su
...
-3. Join the `ManagedClusters` into the `ManagedClusterSet`.
-    
+3. Create a `ManagedClusterAddon` on the managed cluster namespace to deploy the Submariner on the managed cluster.
+
    ```
-   $ oc label managedclusters <managedcluster-name> "cluster.open-cluster-management.io/clusterset=<mangedClusterSet-name>" --overwrite
+   apiVersion: addon.open-cluster-management.io/v1alpha1
+   kind: ManagedClusterAddOn
+   metadata:
+     name: submariner-addon
```
Comment (qiujian16) [inline docs/submarinerAddon.md:41]: the name should be submariner or submariner-addon?

File: docs/submarinerAddon.md [hunk 2]
```diff
@@ -26,19 +26,23 @@ There are some prerequisites for the managed clusters which are going to run `Su
...
+   spec: {}
```
Comment (qiujian16) [inline docs/submarinerAddon.md:42]: what namespace you  are going to install this addon?

File: docs/submarinerAddon.md (hunk 1 above)
Comment (skeeey) [reply docs/submarinerAddon.md:41]: fixed to submariner-addon...

File: docs/submarinerAddon.md (hunk 2 above)
Comment (skeeey) [reply docs/submarinerAddon.md:42]: Create a `ManagedClusterAddon` **on the managed cluster namespace**

Comment (qiujian16) [reply docs/submarinerAddon.md:42]: i mean the installNamespace in spec also.

Comment (skeeey) [reply docs/submarinerAddon.md]: more details are added

Comment (skeeey) [reply docs/submarinerAddon.md]: put the submariner-addon and submariner to the same namespace

**Pull request merged**